src/app/login.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { NextPage } from "next";
+import styles from "./page.module.css";
+
+const LoginPage: NextPage = () => {
```

Comment:
why are you using Nextpage?
the error comes from there.

you can make the page a normal function like this
`const LoginPage = () => {`
this should fix the error